examples/language/llama/benchmark.py

```diff
@@ -242,7 +272,7 @@ def empty_init():
             microbatch_size=args.mbs,
             initial_scale=2**8,
             precision="bf16",
-            overlap_p2p=args.overlap,
+            overlap_p2p=True,
```

Comment:
use args

Reply:
> use args

Add args.overlap in benchmark;
https://github.com/hpcaitech/ColossalAI/pull/6083/commits/982e4ee1f85a3aec285197d8bbdc3ca292bd15ff